frontend/src/components/LeadRow.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { TableRow, TableCell, Box } from '@material-ui/core';
 import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
 import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
 import { useStyles } from './Home';
 
+const SocialMediaLink = ({ lead }) => {
+  const link =
+    lead['facebook'] ||
+    lead['linkedin'] ||
+    lead['twitter'] ||
+    lead['instagram'] ||
+    null;
+
+  return <a href={link}>{link}</a>;
```

Comment:
Can you make this open in a new tab as well?

Reply:
Done!